Sources/Sentry/Public/SentryStacktrace.h

```diff
@@ -21,6 +21,11 @@ SENTRY_NO_INIT
  */
 @property (nonatomic, strong) NSDictionary<NSString *, NSString *> *registers;
 
+/**
+ * Indicates that this stack trace is a snapshot triggered by an external signal.
+ */
+@property (nonatomic, copy) NSNumber *_Nullable snapshot;
```

Comment:
SDK always uses a `NSNumber` for bool, why?

Reply:
When we need it to be nullable is the only reason, otherwise I would use BOOL.

Reply:
it should be nullable, closing it.